site-src/geps/gep-1424.md

```diff
@@ -0,0 +1,384 @@
+
+* NOT already specified as a `parentRef` of another `HTTPRoute` [in the same namespace]
+    * This requirement may be relaxed later to follow the existing [precedence rules](https://gateway-api.sigs.k8s.io/v1alpha2/references/spec/#gateway.networking.k8s.io%2fv1beta1.HTTPRouteRule), but is intended to simplify initial implementation.
+    * The namespace colocation requirement effectively forecloses using this strategy to delegate route management, but there is service mesh [user interest](https://github.com/istio/istio/issues/22997) in merging separately-defined routes together to manage large configurations, multiple versions or canary deployments for a single mesh service.
+* NOT already specified as a `backendRef` of any _other_ `HTTPRoute` with a `Service` `parentRef`
+    * A `Service` MAY still be listed as a `backendRef` on the _same_ `HTTPRoute` where it is specified as a `parentRef`, e.g. to only split off some subset of traffic from the default backend endpoints.
+    * This could be computationally expensive to validate.
+
+This should be enforced by controller validation, which should set an `Accepted: { status: False }` condition with a corresponding new `ParentRefConflict` `RouteConditionReason`.
+
+The logic described here is a bit verbose, but could be validated with conformance tests, and is intended to allow an `HTTPRoute` configuring mesh traffic rules to additionally specify a `Gateway` `parentRef` (or `HTTPRoute` `parentRef` if [GEP-1058: Route Inclusion and Delegation](https://github.com/kubernetes-sigs/gateway-api/pull/1085) moves forward) to apply the same routing rules when exposing a mesh service outside the cluster.
+
+### Drawbacks
+
+* The fact that this pattern is used for mesh configuration is implicit - this may benefit from some additional configuration to map the `HTTPRoute` to a particular mesh implementation rather than being picked up by any or all GAMMA meshes present in a cluster. Possible approaches include:
+* [GEP-1282: Describing Backend Properties](https://gateway-api.sigs.k8s.io/geps/gep-1282/) may be one path to associating a `Service` with a mesh, but likely wouldn't be able to handle the application of multiple `HTTPRoutes` for the same `Service`, but each intended for different mesh implementations
+    * It's currently unclear how relevant this constraint may be, but associating an `HTTPRoute` with a mesh by this method would additionally require an extra graph traversal step.
+* Expecting a `Mesh` `parentRef` or similar reference as proposed in [GEP-1291: Mesh Representation](https://docs.google.com/document/d/1oyA9uUH7pNNxxwy3WZGSWx-edHDBLrujcezr8q3el70/edit#) may be a preferred eventual path forward, but wouldn't be required initially, with the assumption that only one mesh should typically be present in a cluster.
+* No mechanism for egress redirection of traffic from arbitrary hostnames to a mesh service within this approach (but could still be implemented seperately).
+
+## Alternatives
```

Comment:
Agreed, that should be added - it's similar to the nested `parentRefs` approach in functionality, but I like that it inverts it to potentially always have meaningful context (e.g. it could be applicable to "in-mesh" Gateways too)

Reply:
I'm not sure I see the benefit. If a HttpRoute has a Service as parentRef - it is clearly mesh route. What would the extra mesh field in parentRef convey ? 

The Service parentRef provides a clear binding of the route to the service name.

Reply:
The main benefit I can recall would be the ability to disambiguate routes if multiple meshes are ever installed on a cluster.